cmd/soroban-rpc/internal/db/transaction.go

```diff
@@ -18,7 +18,11 @@ import (
 	"github.com/stellar/soroban-rpc/cmd/soroban-rpc/internal/ledgerbucketwindow"
 )
 
-const transactionTableName = "transactions"
+const (
+	transactionTableName              = "transactions"
+	firstLedgerToMigrate              = 2
+	getLedgerRangeMetaArrayCheckValue = 2
```

Comment:
I have seen the use of this variable and it makes the code less readable, please use a nolint instead.

Also, try to define variables in the same file as you use them.

Reply:
Ah the `getLedgerRangeMetaArrayCheckValue ` was originally being used in the transaction.go file and I forgot to remove it.